src/gsTensor/gsTensorDomainIterator.h

```diff
@@ -0,0 +1,225 @@
+/** @file gsTensorDomainIterator.h
```

Comment:
This file should not be here (file moved to folder src/gsDomain in the latest stable)